lib/core/test/unit/Cardano/Pool/MetadataSpec.hs

```diff
@@ -237,3 +321,14 @@ generateWith seed (MkGen gen) =
     --
     -- https://hackage.haskell.org/package/QuickCheck-2.13.2/docs/src/Test.QuickCheck.Gen.html#generate
     size = 30
+
+-- | Count elements in the list matching the given Prism.
+--
+-- >>> count (_Ctor @"MyConstructor") xs
+-- 2
+count :: Prism' s a -> [s] -> Int
```

Comment:
Maybe put this in a tests common module, because other tests which count log messages can use it.

Reply:
c76f29d7b3eb338e26889d9ec41f8ecb37cd08d3